src/components/progress/QueueCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { isFinite } from 'lodash-es';
+import { FormattedMessage } from 'react-intl';
+
+import Paper from '@material-ui/core/Paper';
+
+import Text from '../Text';
+
+function emphasizedH4(chunk) {
+  return (
+    <Text component="strong" variant="h4">
+      {chunk}
+    </Text>
+  );
+}
+
+function emphasizedH5(chunk) {
+  return (
+    <Text component="strong" variant="h5">
+      {chunk}
+    </Text>
+  );
+}
+
+export default function QueueCard({ ahead, style }) {
+  return (
+    <Paper variant="outlined" style={{ padding: 4, ...style }}>
+      <Text
+        variant="caption"
+        component="p"
+        align="center"
+        style={{
+          display: 'flex',
+          flexWrap: 'wrap',
+          alignItems: 'center',
+          justifyContent: 'center',
+          whiteSpace: 'pre-wrap',
+        }}
+      >
+        {isFinite(ahead) ? (
+          <FormattedMessage
+            id="PROGRESS_STATISTICS_QUEUE_POSITION_X"
+            values={{ ahead, focus: emphasizedH4 }}
```

Comment:
Can't say I love the size differences again.

Reply:
id and values can also be passed directly to `<Text>`

Reply:
I found it easier to read if the conditional returned distinct formatted messages rather than doing a conditional earlier to determine the correct id and values and then pass it to Text.

Reply:
The size difference and rich text elements were removed in https://github.com/WildMeOrg/codex-frontend/pull/432/commits/c4f63c1d7012eb1409e1625efe2c9df1d87e388e.